lib/tasks/import.rake

```diff
@@ -0,0 +1,43 @@
+require 'rest-client'
+
+namespace :import do
+  search_url = "https://www.gov.uk/api/search.json?start=0&".freeze
+  batch = 10 #TODO Decide the right batch amount
+
+  desc "import all organisations and respective content items"
+  task all: :environment do
+    Rake::Task["import:organisations"].invoke
+    Rake::Task["import:contentitems"].invoke
+  end
+
+  desc "import all organisations"
+  task organisations: :environment do
+    response_body = RestClient.get("#{search_url}facet_organisations=10000&count=0").body
```

Comment:
I have encapsulated the import for organisation and content_item and use then in the rake task respectively